crates/edr_evm/tests/blockchain.rs

```diff
@@ -380,39 +342,28 @@ async fn test_revert_to_block() {
         // Blocks 1 and 2 are gone
         assert!(blockchain
             .block_by_number(one.header().number)
-            .await
             .unwrap()
             .is_none());
 
         assert!(blockchain
             .block_by_number(two.header().number)
-            .await
             .unwrap()
             .is_none());
 
-        assert!(blockchain
-            .block_by_hash(one.hash())
-            .await
-            .unwrap()
-            .is_none());
-        assert!(blockchain
-            .block_by_hash(two.hash())
-            .await
-            .unwrap()
-            .is_none());
+        assert!(blockchain.block_by_hash(one.hash()).unwrap().is_none());
+        assert!(blockchain.block_by_hash(two.hash()).unwrap().is_none());
     }
 }
 
 #[tokio::test]
```

Comment:
Correct. In order to use blocking threads, which `block_in_place` does, you need to have a multi-threading runtime. This function call always returns a local `u64` stored in `ForkedBlockchain` and `LocalBlockchain` instead of forwarding a call to a remote provider.